cve_bin_tool/cve_scanner.py

```diff
@@ -156,6 +160,12 @@ def get_cves(self, product_info: ProductInfo, triage_data: TriageData):
             # if it fits into both ends of the range, add the cve number
             if passes_start and passes_end:
                 cve_list.append(cve_number)
+                self.all_cve_version_info[cve_number] = VersionInfo(
+                    start_including=versionStartIncluding,
+                    start_excluding=versionStartExcluding,
+                    end_including=versionEndIncluding,
+                    end_excluding=versionEndExcluding,
```

Comment:
While you are here might as well change variable names to Python style

```suggestion
                    start_including=version_start_including,
                    start_excluding=version_start_excluding,
                    end_including=version_end_including,
                    end_excluding=version_end_excluding,
```

Reply:
> I would still rewrite it a bit because if versions check is useless (tuple of empty strings evaluates to True):

yeah, it makes more sense, since we now handle the `"UNKNOWN"` and other `KeyError` before calling `format_version_range` and no empty tuples would be passed. But I would still keep that last `retrun "-"` in case of something.